### ISSUE TITLE
Rethink name

### ISSUE THREAD

**Issue by jonathansimmons** [AsteriskLabs/devise_google_authenticator#41]
The name of this gem implies that it is somehow related to "Google Authenticator". When in reality it simply allows "Two Factor Support" in devise. 

When searching for a gem to help us implement two factor with devise, yours was off my radar because I wanted support for something more than just "Google authenticator". There are many 2 factor apps for both Android and iOS at this point and we don't want our users locked into one app

There is already https://github.com/tinfoil/devise-two-factor. The advantage you have over them is that you've worked out the second step a good bit of people are looking for. https://github.com/tinfoil/devise-two-factor/issues/14

I recommend changing the name of this gem to "devise_two_factor" or "devise_two_factor_athentication" You'll surely see an increase in usage and contribution.

**Comment by vrogers** [comment]
I agree with this, because I almost used another tool until I saw this "issue".

**Comment by jonathansimmons** [comment]
@vrogers you might also look at https://github.com/wmlele/devise-otp. I put some time into that and it's a touch more recently updated.  I found it to better suit my needs.

**Comment by vrogers** [comment]
I like it! thanks!
